tests/MongoDB.Driver.Tests/Specifications/change-streams/ChangeStreamTestRunner.cs

```diff
@@ -419,16 +450,15 @@ private void AssertChangeStreamDocumentPropertyValuesAgainstBackingDocument(Chan
                 fullDocument.Should().BeNull();
             }
 
-            var operationType = actualDocument.OperationType;
-            operationType.ToString().ToLowerInvariant().Should().Be(backingDocument["operationType"].AsString);
-
             var resumeToken = actualDocument.ResumeToken;
             resumeToken.Should().Be(backingDocument["_id"].AsBsonDocument);
 
             var updateDescription = actualDocument.UpdateDescription;
             if (backingDocument.Contains("updateDescription"))
             {
-                updateDescription.Should().Be(backingDocument["updateDescription"].AsBsonDocument);
```

Comment:
As I understand, this implementation has never worked?

Reply:
Doesn't look like it ever worked.

Not sure why it never threw an exception then?

Reply:
Spec:
>This document and its fields only appears if the operationType is update.

So, previously we didn't have `update` operationType.